### PR TITLE
702 support enhanced

### PR DESCRIPTION
**Pull request by christianp86**
Coding for objects SAMC and SAPC updated to avoid syntax errors on 702 systems.

### CODE REVIEW

File: src/zabapgit_object_samc.prog.abap [hunk 1]
```diff
@@ -292,20 +292,24 @@ CLASS lcl_object_samc IMPLEMENTATION.
...
+          objtype        TYPE trobjtype.
 
     objname    = ms_item-obj_name.
     object_key = ms_item-obj_name.
     objtype    = ms_item-obj_type.
 
-    get_persistence( )->lock(
+    IF mo_persistence IS NOT BOUND.
+      get_persistence( ).
+    ENDIF.
```
Comment (larshp) [inline src/zabapgit_object_samc.prog.abap]: perhaps call get_persistence( ) in all cases, inside the get_persistence method there is an IF (note this comment applies for multiple places in this PR)

File: src/zabapgit_object_samc.prog.xml [hunk 2]
```diff
@@ -10,13 +10,6 @@
     <RLOAD>E</RLOAD>
     <UCCHECK>X</UCCHECK>
    </PROGDIR>
-   <TPOOL>
-    <item>
-     <ID>R</ID>
-     <ENTRY>Include ZABAPGIT_OBJECT_SAMC</ENTRY>
-     <LENGTH>28</LENGTH>
-    </item>
-   </TPOOL>
```
Comment (larshp) [inline src/zabapgit_object_samc.prog.xml]: Looks like the include description was lost, if you enter it again manually and push it should be okay(also relevant for the other include)

File: src/zabapgit_object_samc.prog.abap (hunk 1 above)
Comment (christianp86) [inline src/zabapgit_object_samc.prog.abap]: I agree :)
Updated the code

**Comment by larshp** [comment]
PR related to #789

**Pull request merged**